src/views/Run/internal/ResultDisplay/internal/ReportSection/ReportSection.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC } from 'react'
+
+import { RunReport } from '@/benchttp'
+import { isDurationMetricField } from '@/benchttp/metrics'
+import { Tag, Typography } from '@/components'
+import { convertNsToMs } from '@/tools/converters'
+import { parseMilliseconds } from '@/tools/parsers'
+
+import { TestResult } from './internal/components'
+import { getFailedTestCount } from './internal/ReportSection.helpers'
+
+export interface IProps {
+  report: RunReport
+}
+
+export const ReportSection: FC<IProps> = ({ report }) => {
+  const failedTestCount: number = getFailedTestCount(report.tests.results)
+
+  return (
+    <div>
+      <div className="f f-ai-center mb-4">
+        <Typography element="h1">Status : </Typography>
+        <Tag text={`Test ${report.tests.pass ? 'passed' : 'failed'}`} />
+      </div>
+      <Typography element="p" size="h4" className="mb-4">
+        {failedTestCount}/{report.tests.results.length} tests{' '}
```

Comment:
```suggestion
        {report.tests.pass ? report.tests.results.length : failedTestCount}/
        {report.tests.results.length} tests{' '}
```

Reply:
Else, we have always the number of failed tests even though we want the successful ones when all tests have passed.